lib/main.dart

```diff
@@ -1,125 +1,59 @@
-    //
-    // The Flutter framework has been optimized to make rerunning build methods
-    // fast, so that you can just rebuild anything that needs updating rather
-    // than having to individually change instances of widgets.
-    return Scaffold(
-      appBar: AppBar(
-        // TRY THIS: Try changing the color here to a specific color (to
-        // Colors.amber, perhaps?) and trigger a hot reload to see the AppBar
-        // change color while the other colors stay the same.
-        backgroundColor: Theme.of(context).colorScheme.inversePrimary,
-        // Here we take the value from the MyHomePage object that was created by
-        // the App.build method, and use it to set our appbar title.
-        title: Text(widget.title),
-      ),
-      body: Center(
-        // Center is a layout widget. It takes a single child and positions it
-        // in the middle of the parent.
-        child: Column(
-          // Column is also a layout widget. It takes a list of children and
-          // arranges them vertically. By default, it sizes itself to fit its
-          // children horizontally, and tries to be as tall as its parent.
-          //
-          // Column has various properties to control how it sizes itself and
-          // how it positions its children. Here we use mainAxisAlignment to
-          // center the children vertically; the main axis here is the vertical
-          // axis because Columns are vertical (the cross axis would be
-          // horizontal).
-          //
-          // TRY THIS: Invoke "debug painting" (choose the "Toggle Debug Paint"
-          // action in the IDE, or press "p" in the console), to see the
-          // wireframe for each widget.
-          mainAxisAlignment: MainAxisAlignment.center,
-          children: <Widget>[
-            const Text(
-              'You have pushed the button this many times:',
-            ),
-            Text(
-              '$_counter',
-              style: Theme.of(context).textTheme.headlineMedium,
-            ),
-          ],
+    return SafeArea(
+      child: Scaffold(
+        backgroundColor: UIConstants.backgroundColor,
+        body: SingleChildScrollView(
+          child: Column(
+            children: [
+              MovieHeader(
+                movie: movie,
+              ),
+              Padding(
+                padding: const EdgeInsets.all(
+                  UIConstants.columnPadding,
+                ),
+                child: Column(
+                  mainAxisAlignment: MainAxisAlignment.start,
+                  crossAxisAlignment: CrossAxisAlignment.start,
+                  children: <Widget>[
+                    const ButtonsBar(),
+                    UIConstants.sectionSpace,
+                    MovieOverview(
+                      movie: movie,
+                    ),
+                    UIConstants.sectionSpace,
+                    MovieInfo(
+                      movie: movie,
```

Comment:
instead of passing the whole movie as a parameter, you should send the specific info needed

Reply:
In fact, run the auto-format for the whole project to fix the imports order as well. You can click the lib folder in the IDE and then use the shortcut.

Reply:
done 
I did the same on movie_header and movie_info (not sure if that was intended as well).